db/dev_seeds/polls.rb

```diff
@@ -28,19 +28,49 @@
               ends_at:   1.month.ago,
               results_enabled: true,
               stats_enabled: true)
+
+  Poll.all.each do |poll|
```

Comment:
Avoid using `Poll.all.each` as it can exceed the amount of memory available if said model has an associated table with a considerable amount of records, as shown [here](https://guides.rubyonrails.org/v4.2/active_record_querying.html#retrieving-multiple-objects-in-batches)

Reply:
In this case is not a problem. This is the seeds file and we don't have thousands of records, but anyway I will update it. It's never a bad idea to use the "Best Practices" 😉

Reply:
Resolved as per commit https://github.com/consul/consul/pull/2833/commits/723eacff0f412bc1d7641c3db1106dedccda6dde